### DISCUSSION TITLE
Change tool strip menu item background color when hovered

### DISCUSSION THREAD

**Discussion by MadProbe** [dotnet/winforms#8667] [Q&A]
Is there a way to change this, I want to make dark theme for application and this on-hover background makes it unavailable to see white text. I have searched the internet and found **no** solution for this.

**Comment by dreddy-work** [comment]
Can you add a screenshot of what you mean by 
>and this on-hover background makes it unavailable to see white text

**Comment by MadProbe** [comment]
![image](https://user-images.githubusercontent.com/49519179/225412262-6bf0f369-7558-4832-be8a-5739a949210e.png)
On image there are 2 buttons, left button is button when I hovered on it by mouse and right is untouched.
